src/components/Map/BannerContent.tsx

```diff
@@ -0,0 +1,194 @@
+  position: fixed;
+  bottom: -740px;
+  z-index: 1;
+  height: 900px;
+  background-color: #FFFFFF;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  cursor: grab;
+
+  &:active {
+    cursor: grabbing;
+  }
+`;
+
+const BottomSheetContent = styled.div`
+  overflow: auto;
+  width: 100%;
+  height: 68%;
+`;
+
+const BottomSheetHeader = styled.div`
+    border: 0px;
+    background-color: #BBC7D3;
+    border-radius: 12px;
+    width: 60px;
+    height: 6.747px;
+    margin: 25px;
+    padding-top: 4px;
+
+`;
+
+const BottomSheetFilter = styled.div`
+    overflow: scroll;
+    width: 100%;
+    height: 60px;
+    display: flex;
+    margin-top: 5px;
+    margin-bottom: 5px;
+
+    button {
+      height: 43px;
+      border: 0px;
+      border-radius: 30px;
+      font-size: 13px;
+      font-style: normal;
+      font-weight: 510;
+      line-height: normal;
+      letter-spacing: -0.2px;
+      margin-left: 0.5rem;
+      margin-right: 0.5rem;
+      box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.12);
+      height: 45px;
+      cursor: pointer;
+      background-color: #FFFFFF;
+      border: 1px solid #d4d3d3;
+      color: #7e7d7d;
+    }
+    
+    .clickedDay {
+      background-color: #FFFFFF;
+      border: 1px solid #FB7876;
+      color: #FB7876;
+    }
+
+    .clickedCategory {
+      background-color: #FFFFFF;
+      border: 1px solid #0199FF;
+      color: #0199FF;
+    }
+`;
+
+const DayFilterContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  width: 40%;
+
+  button {
+    width: 45px;
+  }
+
+`;
+
+const CategoryFilterContanier = styled.div`
+  display: flex;
+  justify-content: space-around;
+  width: 60%;
+
+  button {
+    width: 90px;
+  }
+`;
+
+type BottomSheetProps = {
+  setSelectedDay: (value: string) => void;
+  selectedDay: string;
+  setSelectedCategory: (value: string) => void;
+  selectedCategory: string;
+  booths: Booth[];
+}
+
+export default function BottomSheet({
+  setSelectedDay,
+  selectedDay,
+  setSelectedCategory,
+  selectedCategory,
+  booths,
+}: BottomSheetProps) {
+  const { sheet, content } = useBottomSheet();
+  const [isSwipe, setIsSwipe] = useState<boolean>(false);
+
+  const categories = useFetchCategories();
+  const { days, filters } = categories;
+
+  const handleClick = () => {
+    setIsSwipe(true);
+  };
+
+  const handleSetFilterDay = (category: string) => {
+    setSelectedDay(category);
+  };
+
+  const handleSetFilterCategory = (category: string) => {
+    setSelectedCategory(category);
+  };
+
+  console.log(booths);
+  return (
+    <Wrapper
+      ref={sheet}
+      onClick={handleClick}
+      className={isSwipe ? 'active' : ''}
+    >
+      <BottomSheetHeader />
+      <BottomSheetContent ref={content}>
+        <BottomSheetFilter>
+          {days && days.map((category: string) => (
```

Comment:
이렇게 사용이 되게끔 days 데이터를 받아오는쪽에서 타입이 정의되는게 좋을 것 같아요~
```suggestion
          {days && days.map((category) => (
```